DirectEthernet/src/main/pack/DirectEthernet.ini

```diff
@@ -1,6 +1,6 @@
 [info]
 name = Scala Direct Ethernet
-version = 1.0.0-M1
```

Comment:
Might be nice to have this brought in from a single point both here and in `build.sbt`.